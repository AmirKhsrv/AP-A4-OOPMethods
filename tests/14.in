show_author_info 1342
show_sorted_shelf 99 read Fantasy
show_author_info 1413
credit 88
credit 70
