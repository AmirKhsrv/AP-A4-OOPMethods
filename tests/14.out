id: 1342
Name: Alan Lightman
Year of Birth: 1976
Place of Birth: Armenia
Member Since: March 15 2015
Genres: Fiction, Science Fiction
Books:
id: 354
Title: Awaken Me
Genre: Fantasy
Author: E.M. Sky
***
id: 91
Title: Farsighted
Genre: Fantasy
Author: Emlyn Chand
***
id: 194
Title: Relics of Camelot
Genre: Fantasy
Author: L.H. Nicole
***
id: 28
Title: Charged
Genre: Romance
Author: Jay Crownover
***
id: 222
Title: Everything Is Illuminated
Genre: Fiction
Author: Jonathan Safran Foer
***
id: 203
Title: Here be Dragons
Genre: Historical
Author: Sharon Kay Penman
***
id: 190
Title: How to Win Friends and Influence People
Genre: Nonfiction
Author: Dale Carnegie
***
id: 323
Title: Joshua
Genre: Fiction
Author: Joseph F. Girzone
***
id: 238
Title: Lamb: The Gospel According to Biff Christ's Childhood Pal
Genre: Humor
Author: Christopher Moore
***
id: 56
Title: Man's Search for Meaning
Genre: Nonfiction
Author: Viktor E. Frankl
***
id: 274
Title: Otherwise Alone
Genre: Romance
Author: Shay Savage
***
id: 283
Title: Sacajawea
Genre: Historical
Author: Anna Lee Waldo
***
id: 89
Title: Season of Migration to the North
Genre: Fiction
Author: Tayeb Salih
***
id: 217
Title: Space Viking
Genre: Science Fiction
Author: H. Beam Piper
***
id: 87
Title: Yes Means Yes!: Visions of Female Sexual Power and A World Without Rape
Genre: Feminism
Author: Jaclyn Friedman
***
id: 253
Title: You Belong to Me
Genre: Romance
Author: Karen      Rose
***
id: 1413
Name: Kimber S. Dawn
Year of Birth: 1990
Place of Birth: Israel
Member Since: March 21 2013
Genres: Romance
Books:
0.009241
0.006953
